setup/www/tasks/site-setup.yaml

```diff
@@ -81,3 +84,16 @@
     - "/home/nodejs/build-site.sh nodejs"
     - "/home/nodejs/build-site.sh iojs"
   tags: setup
+
+- name: Site Setup | Add periodic tasks to crontab
+  lineinfile:
+    dest: /etc/crontab
+    line: "{{ item }}"
+  with_items:
+    - '*/5 *   * * *   dist    /home/staging/tools/promote/promote_nightly.sh nodejs'
+    - '*/5 *   * * *   dist    /home/staging/tools/promote/promote_nightly.sh iojs'
+    - '*  *    * * *   nodejs  /home/nodejs/check-build-site.sh nodejs'
+    - '*  *    * * *   nodejs  /home/nodejs/check-build-site.sh iojs'
+    - '*  *    * * *   root    /home/nodejs/cdn-purge.sh'
+    - '0 */4   * * *   nodejs  rsync -avz --delete benchmark:charts/ /home/www/benchmarking/charts/'
```

Comment:
perhaps add a nightly cronjob that updates the npm packages? similar to https://github.com/jbergstroem/build/blob/feature/refactor-the-world/ansible/playbooks/jenkins/linter.yml#L36..L40.